luanderfarias.md

```diff
@@ -30,12 +29,16 @@ Simplo is developed based on other open-source vr headsets out there. Here are s
 
 ## Budget
 
+## Headset
+
 | Product         | Supplier/Link                         | Cost   |
 | --------------- | ------------------------------------- | ------ |
-| 1x Arduino Micro | https://store-usa.arduino.cc/products/arduino-micro | $38.61 |
-| 1x Acelerometer and gyroscope mpu6050 | https://www.ebay.com/itm/295005468005?chn=ps&mkevt=1&mkcid=28&srsltid=AeTuncrx8Nd6PKD56swU6q1U-7z0kxZe6F_kvLrvcyVwZBaKHMdjDY9cypM | $0.99 |
-| 1x 5-inch Raspberry Pi LCD Screen with HDMI interface | https://www.crowpi.cc/products/rr050-5-inch-raspberry-pi-screen-touchscreen-monitor-tft-display-for-raspberry-pi-4b-3b-3b-2b-bb-black-banana-pi-windows-10-8-7?variant=39701621014661&currency=USD&utm_medium=product_sync&utm_source=google&utm_content=sag_organic&utm_campaign=sag_organic&srsltid=AeTuncr8B4F9Q7yh7FjVHqnMsxJRtYMvFLyGgjXL76QPDOjYqyzIV27kEU8 | $53.10 |
-| 4x Arduino camera module | https://tinkersphere.com/arduino-compatible-components/944-ov7670-vga-camera-module-for-arduino.html | $44.95 |
+| 1x Arduino Micro | https://store-usa.arduino.cc/products/arduino-micro | $38.51 |
+| 1x mpu9250 | https://pt.aliexpress.com/item/1005004439490641.html?pdp_npi=2%40dis%21BRL%21R%24%2040%2C57%21R%24%2038%2C94%21%21%21%21%21%402101d1b616734837560467175eee53%2112000029185573027%21btf&_t=pvid%3A3bc2cc48-d54b-4dd2-8a53-a41f212935b8&afTraceInfo=1005004439490641__pc__pcBridgePPC__xxxxxx__1673483756&spm=a2g0o.ppclist.product.mainProduct&gatewayAdapt=glo2bra | $11.29 |
+| 2x Fresnel-lens | https://pt.aliexpress.com/item/33029909783.html?gatewayAdapt=glo2bra | $8.46 |
+| 1x HDMI VR Display | https://pt.aliexpress.com/item/32981354673.html?gatewayAdapt=glo2bra | $30.35 |
```

Comment:
hiii @LuanderFarias it looks like the VR display you linked says it only supports computer & rpi?
<img width="741" alt="Screenshot 2023-01-12 at 8 20 24 PM" src="https://user-images.githubusercontent.com/65808924/212215169-bf8c776a-1b91-4b22-8699-db7d791f0eb6.png">

Reply:
yeah. Whats the problem?

Reply:
your project is arduino based?